forms/viewsets/form.py

```diff
@@ -45,3 +50,9 @@ def get_permissions(self):
                 IsAuthenticated(),
             ]
         return super().get_permissions()
+
+
+class AttachmentViewSet(viewsets.ModelViewSet):
```

Comment:
GET:
- I changed the `user_id` of a file I uploaded and noticed it was still visible here. Sure, I was logged in as an user with permissions on the administrative site, but considering the below queryset, I think this would always return all attachments even if that wasn't the case. It should only return ones uploaded by the active user.
- The attachment ID(s) aren't included in the responses. I think the returned link in the `attachment` field is also bogus, but that can simply be ignored.
- The ability to filter out attachments that are already attached to an answer will become important really soon, but this works as the first version.

DELETE:
- Doesn't seem to do what it's supposed to: such a request returns the attachment data just like a GET and the attachment doesn't get deleted.